src/main/java/fr/openent/competences/service/impl/DefaultUtilsService.java

```diff
@@ -192,6 +192,17 @@ public void getAllMultiTeachers(final String structureId, final JsonArray groupI
         }));
     }
 
+    public Future<JsonArray> getAllMultiTeachers(final String structureId, final JsonArray groupIds){
+        Promise<JsonArray> promiseMultiTeachers = Promise.promise();
+        JsonObject action = new JsonObject()
+                .put("action", "multiTeaching.getAllMultiteachers")
+                .put("structureId", structureId)
+                .put("groupIds", groupIds);
+        eb.request(Competences.VIESCO_BUS_ADDRESS, action, DELIVERY_OPTIONS,
+                FutureHelper.getMessageJsonArray(promiseMultiTeachers, "[DefaultUtilsService] : getAllMultiteachers : "));
+        return promiseMultiTeachers.future();
+    }
+
```

Comment:
Même principe par rapport à mon commentaire au-dessus : utilise la méthode déjà existante pour polymorpher ta méthode `getAllMultiTeachers ` que tu veux faire retourner une future

Exemple : 

```suggestion
public Future<JsonArray> getAllMultiTeachers(final String structureId, final JsonArray groupIds){
        Promise<JsonArray> promiseMultiTeachers = Promise.promise();
        getAllMultiTeachers(structureId, groupIds, event -> {
            if (event.isLeft()) {
                log.error(String.format("[Competences@%s::getAllMultiTeachers] Error during sql request: %s.",
                        this.getClass().getSimpleName(), event.left().getValue()));
                promiseMultiTeachers.fail(event.left().getValue());
            } else {
                promiseMultiTeachers.complete(event.right().getValue());
            }
        });
        return promiseMultiTeachers.future();
    }
```